v2/schema/context.jsonld

```diff
@@ -0,0 +1,132 @@
+{
+  "@context": {
+    "dcat": "http://www.w3.org/ns/dcat#",
+    "@vocab": "http://www.w3.org/ns/dcat#",
+    "dc": "http://purl.org/dc/terms/",
+    "dctypes": "http://purl.org/dc/dcmitype/",
+    "gbl": "http://geoblacklight.org/v2/schema#",
+    "ogc": "http://www.opengis.net/ogc#",
+    "pod": "https://project-open-data.cio.gov/v1.1/schema#",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+    "accrualPeriodicity": {
+      "@type": "@id",
+      "@id": "dc:accrualPeriodicity"
+    },
+    "accessLevel": {
+      "@type": "@id",
+      "@id": "pod:accessLevel"
+    },
+    "accessURL": {
+      "@type": "@id",
+      "@id": "dcat:accessURL"
+    },
+    "conformsTo": {
+      "@type": "@id",
+      "@id": "dc:conformsTo"
+    },
+    "creator": {
+      "@type": "@id",
+      "@id": "dc:creator"
+    },
+    "describedBy": {
+      "@id": "http://www.w3.org/2007/05/powder#describedby",
+      "@type": "@id"
+    },
+    "describedByType": {
+      "@type": "@id",
+      "@id": "pod:describedByType"
+    },
+    "description": {
+      "@type": "@id",
+      "@id": "dc:description"
+    },
+    "downloadURL": {
+      "@type": "@id",
+      "@id": "dcat:downloadURL"
+    },
+    "format": {
+      "@type": "@id",
+      "@id": "dctypes:format"
+    },
+    "geom": {
+      "@type": "@id",
+      "@id": "ogc:envelope"
+    },
+    "geomType": {
+      "@type": "@id",
+      "@id": "gbl:geomType"
+    },
+    "isPartOf": {
+      "@type": "@id",
+      "@id": "dc:isPartOf"
+    },
+    "issued": {
+      "@type": "@id",
+      "@id": "pod:issued"
+    },
+    "language": {
+      "@type": "@id",
+      "@id": "dc:language"
+    },
+    "layerId": {
+      "@type": "@id",
+      "@id": "dc:identifier"
+    },
+    "license": {
+      "@type": "@id",
+      "@id": "dc:license"
+    },
+    "mediaType": {
+      "@type": "@id",
+      "@id": "dcat:mediaType"
+    },
+    "modified": {
+      "@type": "@id",
+      "@id": "pod:modified"
+    },
+    "publisher": {
+      "@type": "@id",
+      "@id": "dc:publisher"
+    },
+    "provenance": {
+      "@type": "@id",
+      "@id": "dc:provenance"
+    },
+    "resourceType": {
+      "@type": "http://purl.org/dc/dcmitype/",
+      "@id": "dc:type"
+    },
+    "rights": {
+      "@type": "@id",
+      "@id": "dc:rights"
+    },
```

Comment:
DCAT rights are rights statements. Free text it looks like. This schema is based on an enhanced DCAT called [Project Open Data Metadata Schema](https://project-open-data.cio.gov/v1.1/schema/). which adds the  `accessLevel` property.